static/src/javascripts/bootstraps/standard/atoms.js

```diff
@@ -26,6 +26,42 @@ const bootstrapAtom = <A>(atomMaker: AtomMaker<A>, atomType: AtomType) => {
     });
 };
 
+const initCharts = () => {
+    const iframes: HTMLIFrameElement[] = ([
+        ...document.querySelectorAll('.atom--chart > .atom__iframe'),
+    ]: any);
+
+    window.addEventListener('message', event => {
+        const iframe: ?HTMLIFrameElement = iframes.find(i => {
+            try {
+                return i.name === event.source.name;
+            } catch (e) {
+                return false;
+            }
+        });
+        if (iframe) {
+            try {
+                const message = JSON.parse(event.data);
+                switch (message.type) {
+                    case 'set-height':
+                        iframe.height = message.value;
+                        break;
+                    default:
+                }
+                // eslint-disable-next-line no-empty
+            } catch (e) {}
+        }
+    });
+
+    iframes.forEach(iframe => {
+        const src = (iframe.getAttribute('srcdoc') || '').replace(
+            /gu-script/g,
```

Comment:
Is it worth making sure we only replace these values if they are inside `<` and `/>`?

Reply:
Yeah probably 😅 Will do.